guides/common/assembly_performing-additional-configuration-on-capsule-server.adoc

```diff
@@ -27,5 +27,9 @@ include::modules/proc_enabling-power-management-on-managed-hosts.adoc[leveloffse
 // Configuring DNS, DHCP, and TFTP on {SmartProxyServer}
 include::modules/proc_configuring-dns-dhcp-and-tftp.adoc[leveloffset=+1]
 
+include:modules/proc_configuring-dhcp-on-smart-proxies.adoc[leveloffset=+2]
+
+include:modules/proc_configuring-dns-on-smart-proxies.adoc[leveloffset=+2]
```

Comment:
I still think these are reference guides, not to be consumed by most users since they don't use the installer. We have `guides/common/assembly_using-infoblox-as-dhcp-and-dns-providers.adoc` and `guides/common/modules/proc_configuring-dns-dhcp-and-tftp.adoc` as more generic instructions. Ideally we expand those with the additional providers.

If you want, I can work with you to make that happen.

Reply:
I've split the procedures.